pkg/volume/csi/csi_mounter.go

```diff
@@ -259,6 +260,11 @@ func (c *csiMountMgr) SetUpAt(dir string, mounterArgs volume.MounterArgs) error
 		return errors.New(log("mounter.SetupAt failed: %v", err))
 	}
 
+	c.supportsSELinux, err = c.kubeVolHost.GetHostUtil().GetSELinuxSupport(dir)
+	if err != nil {
+		klog.V(2).Info(log("error checking for SELinux support: %s", err))
```

Comment:
should we return error instead?  I guess that would also introduce the problem that we wouldn't try cleaning up the mount since asw won't be marked as mounted. cc @jingxu97

Reply:
I wasn't really sure what to do about it. The old implementation just assumed `false` and logged the error. I suppose moving the SELinux check to here does raise the possibility of backing out in the same way as failing during the fsgroup change a few lines below.

The question is whether we **should** fail the mount if we can't detect the SELinux status... I could go either way on that.

Reply:
It might be problematic to fail the mount if selinux support can't be determined. I would vote for keeping existing behaviour and assume selinux is not supported if we can't determine it.

Reply:
It seems like the error handling below is also problematic. If applyFSGroup returns error and rolling back the mount failed, then we may also get into a bad state. But I guess that is less likely to happen in practice.

Reply:
> If applyFSGroup returns error and rolling back the mount failed, then we may also get into a bad state. But I guess that is less likely to happen in practice.

This is a separate issue that could be fixed by returning uncertain mount status that is @gnufied working on.